tools/migration/ppaas-artifact-converter/src/main/java/org/apache/stratos/manager/dto/Cartridge.java

```diff
@@ -0,0 +1,277 @@
+package org.apache.stratos.manager.dto;
+
+import org.apache.stratos.manager.dao.*;
```

Comment:
please avoid importing on demand. That means, avoid importing blah.blah.*; Always import what is needed.